### PR TITLE
Update reading-files.md

### PR DESCRIPTION
**Pull request by rcseacord**
The smaller example was not going to compile because kPath was not defined.

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

File: docs/wiki/development/reading-files.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ In order to internalize the main API, consider the same example without error ch
 int main(int argc, char* argv[]) {
   std::string content;
   osquery::readFile("/foo/bar.txt", content);
-  std::cout << "Contents of " << kPath << ":\n" << content;
+  std::cout << "Contents of " << "/foo/bar.txt" << ":\n" << content;
```
Comment (marpaia) [inline docs/wiki/development/reading-files.md]: Nice catch, however, if this is being changed into a string, all of those stream operators are no longer necessary. Consider:

```cpp
std::cout << "Contents of /foo/bar.txt:\n" << content;
```

Comment (rcseacord) [inline docs/wiki/development/reading-files.md]: sure, guess I could have simplified that further on my own.  8^)

**Comment by theopolis** [comment]
ok to test

**Comment by theopolis** [comment]
You can avoid the merges with `git pull --rebase origin master`. We are very resistant to merging branches that contain merges. To fix this you can reset to `54dfb52

**Comment by theopolis** [comment]
It looks like the updates are replicated in #1585. Assuming that supersedes this PR?

**Pull request closed without merging**